src/native/windows/apps/prunsrv/prunsrv.c

```diff
@@ -1758,9 +1763,6 @@ void __cdecl main(int argc, char **argv)
         }
     }
 
-    apxLogOpen(gPool, SO_LOGPATH, SO_LOGPREFIX, SO_LOGROTATE);
-    apxLogLevelSetW(NULL, SO_LOGLEVEL);
-    apxLogWrite(APXLOG_MARK_DEBUG "Apache Commons Daemon procrun log initialized.");
     if (SO_LOGROTATE)
         apxLogWrite(APXLOG_MARK_DEBUG "Log will rotate each %d seconds.", SO_LOGROTATE);
 
```

Comment:
I'd include these two lines in the code you move as well. It think it makes sense to keep this message with the "log initialized message"

Reply:
Done.